### ISSUE TITLE
[Feature Request] Static Builds or Makefile equivalent

### ISSUE THREAD

**Issue by sinandredemption** [johannesCmayer/bepl#6]
It would be convenient to just download/make/install the static builds instead of manually installing all the dependencies through conda and then running the program through command-line.

**Comment by johannesCmayer** [comment]
Agreed. Though, I am not using this project right now, and I don't have time to look into this. Feel free to PR or Fork.
